cf/mixin/fielddomainlist.py

```diff
@@ -0,0 +1,339 @@
+from ..functions import _DEPRECATION_ERROR_ARG
+
+
+class FieldDomainList:
+    """TODO."""
+
+    def select_by_construct(self, *identities, OR=False, **conditions):
+        """Select elements by their metadata constructs.
+
+        To find the inverse of the selection, use a list comprehension
+        with the !match_by_construct` method of the construct
```

Comment:
```suggestion
        with the `match_by_construct` method of the construct
```

Reply:
Or possibly `` `! `` ...

Reply:
back-tick bang is right (I struggled to get the markdown right light you did!)

Reply:
I did have to look up how to render the markdown there!  FYI, it turns out that if you do double ticks and leave a space between them and the input, you can escape the back-ticks, whereas black-slash escaping doesn't seem to work, e.g.  `'' he'llo ''` but replacing `'` with `` ` ``.